### PR TITLE
Add docs on Address::fromString

### PR DESCRIPTION
**Pull request by gisostallenberg**
See https://github.com/symfony/symfony/issues/33086

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -158,6 +158,15 @@ Multiple addresses are defined with the ``addXXX()`` methods::
         // ...
     ;
 
+A NamedAddress can be created from a string
```
Comment (OskarStark) [inline mailer.rst]:
```suggestion
A ``NamedAddress`` can be created from a single string::
```

File: mailer.rst [hunk 2]
```diff
@@ -158,6 +158,15 @@ Multiple addresses are defined with the ``addXXX()`` methods::
...
+
+    $email = (new Email())
+    // ...
```
Comment (OskarStark) [inline mailer.rst]:
```suggestion

```

File: mailer.rst [hunk 3]
```diff
@@ -158,6 +158,15 @@ Multiple addresses are defined with the ``addXXX()`` methods::
...
+    ->from(NamedAddress::fromString('Fabien <fabien@example.com>'))
+    //
```
Comment (OskarStark) [inline mailer.rst]:
```suggestion
    // ...
```

File: mailer.rst [hunk 4]
```diff
@@ -158,6 +158,15 @@ Multiple addresses are defined with the ``addXXX()`` methods::
         // ...
     ;
 
+A NamedAddress can be created from a string
+
+    $email = (new Email())
+    // ...
+    ->from(NamedAddress::fromString('Fabien <fabien@example.com>'))
```
Comment (OskarStark) [inline mailer.rst]:
```suggestion
        ->from(NamedAddress::fromString('Fabien <fabien@example.com>'))
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Nice addition, can you please apply my suggestions?

**Comment by gisostallenberg** [comment]
> Nice addition, can you please apply my suggestions?

Sure, done

File: mailer.rst [hunk 5]
```diff
@@ -158,6 +158,14 @@ Multiple addresses are defined with the ``addXXX()`` methods::
         // ...
     ;
 
+A ``NamedAddress`` can be created from a single string::
+
+    $email = (new Email())
+        ->from(NamedAddress::fromString('Fabien <fabien@example.com>'))
+    // ...
```
Comment (OskarStark) [inline mailer.rst]:
```suggestion
        // ...
```

Comment (OskarStark) [reply mailer.rst]: just one minor comment :)

**Comment by gisostallenberg** [comment]
@OskarStark updated this according to https://github.com/symfony/symfony/pull/33091

**Comment by OskarStark** [comment]
Can you please apply my comment and fix the one mentioned by `DOCtor-RST`? Thanks.

**Comment by gisostallenberg** [comment]
> Can you please apply my comment and fix the one mentioned by `DOCtor-RST`? Thanks.

@OskarStark done

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Giso, thanks for adding the docs for this feature that you also contributed. Congrats on your first Symfony Docs contribution!

**Pull request merged**